spec/string_spec.rb

```diff
@@ -29,4 +31,35 @@
     end
   end
 
+  it "replaces spaces with underscores" do
+    Benchmark.ips do |x|
+      str = Random.new.bytes(1024)
```

Comment:
yeah you'd probably want to have some of those characters.

I wonder if Faker is a better choice for this